src/resources/GameResources.py

```diff
@@ -1,27 +1,34 @@
+from ..LevelSelector import LevelSelector
 from .entities.AbstractDungeonEntity import AbstractDungeonEntity
 from .entities.character import Character
 from .entities.ColorChanger import ColorChanger
 from .entities.EnemyManager import EnemyManager
-from .entities.level.Level import Level
 
 
 class GameResources:
     """Holds objects that are used for during game runtime"""
 
-    def __init__(self, testing: bool):
-        self.level = Level(20, 15, [1, 2, 3, 4], [])
+    def __init__(self, testing: bool, bless: bool):
+        self.level_selector = LevelSelector()
+
+        self.level = self.level_selector.create_level()
         self.player = Character(symbol="$", x=self.level.width // 2, y=self.level.height // 2)
```

Comment:
When a level is created can we ensure that there will be no enemey RIGHT in front of the player when a new one is created?

Reply:
New enemies are no created yet. It just leaves the enemies from the last level in place. We sill need to talk about how enemy spawning will work in the final product.